lib/tokenize.cpp

```diff
@@ -7937,6 +7939,56 @@ bool Tokenizer::simplifyRedundantParentheses()
     return ret;
 }
 
+void Tokenizer::simplifyTypeIntrinsics()
+{
+    static std::unordered_map<std::string, std::string> intrinsics = {
```

Comment:
It seems this function is used for C as well as for C++?
Could that map be `const`? Actually all statics should be anyway to ensure thread-safety.

Reply:
Fixed.